scripts/generate/jobs.mjs

```diff
@@ -0,0 +1,456 @@
+import { exec as execSync } from 'child_process';
+import { existsSync } from 'fs';
+import { mkdir, writeFile } from 'fs/promises';
+import { resolve } from 'path';
+import { promisify } from 'util';
+
+const exec = promisify(execSync);
+
+export function resolvePath(name, ...args) {
+  return resolve(process.cwd(), 'packages', name, ...args);
+}
+
+export const jobs = [
+  {
+    description: 'Preliminary Generation Check',
+    callback: ({ name }) =>
+      new Promise((resolve, reject) =>
+        existsSync(resolvePath(name)) ? reject(new Error('A package with the given name already exists')) : resolve('')
+      )
+  },
+  {
+    description: 'Workspace Folder Creation',
+    callback: async ({ name, description, umd }) => {
+      await mkdir(resolvePath(name));
+      await writeFile(
+        resolvePath(name, 'package.json'),
+        JSON.stringify(
+          umd
+            ? {
+                name: `@joshdb/${name}`,
+                version: '1.0.0',
+                description,
+                author: 'Évelyne Lachance <eslachance@gmail.com> (https://evie.codes/)',
+                contributors: [],
+                license: 'Apache-2.0',
+                main: 'dist/index.js',
+                module: 'dist/index.mjs',
+                browser: 'dist/index.umd.js',
+                unpkg: 'dist/index.umd.js',
+                types: 'dist/index.d.ts',
+                exports: {
+                  import: './dist/index.mjs',
+                  require: './dist/index.js',
+                  types: './dist/index.d.ts'
+                },
+                sideEffects: false,
+                scripts: {
+                  test: 'jest',
+                  lint: 'eslint src tests --ext ts --fix -c ../../.eslintrc',
+                  build: 'rollup -c rollup.config.ts',
+                  prepack: 'rollup-type-bundler',
+                  bump: 'cliff-jumper',
+                  'check-update': 'cliff-jumper --dry-run'
+                },
+                dependencies: {},
+                devDependencies: {
+                  '@favware/rollup-type-bundler': '^1.0.7'
```

Comment:
Could these dependencies be taken from an existing package.json in order to keep the version up to date?

Reply:
They potentially could, but either way we can just update this sometimes if it's required, but it's only one package, so I'm not very worried.